common/history/power_blocs/imperia_power_blocs.txt

```diff
@@ -0,0 +1,23 @@
+﻿POWER_BLOCS = {
+	c:CHI = {
+		create_power_bloc = {
+			name = MANDATE_OF_HEAVEN
+			map_color = { 252 185 61 }
+
+			founding_date = 1644.24.12 # Fall of Beijing to the Manchus
+			identity = identity_sovereign_empire
+			principle = principle_vassalization_2
+
+			# subjects are automatically part of the bloc
+		}
+		if = {
+			limit = {
+				has_dlc_feature = power_bloc_features
+			}
+			power_bloc = {
+				add_principle = principle_internal_trade_2
```

Comment:
This will cause errors and issues, as internal_trade requires the Trade League Power Bloc. Please swap it to market_unification.